packages/playwright/src/reporter.ts

```diff
@@ -66,12 +66,18 @@ function mapTestStepHook(step: TestStep): "beforeEach" | "afterEach" | undefined
   }
 }
 
-interface ReplayPlaywrightConfig extends ReplayReporterConfig {
+type ReplayPlaywrightRecordingMetadata = {
```

Comment:
I'm open to suggestions for a naming scheme of those things, this one is already a mouthful 😅

Reply:
I think this name is okay. We could shorten it to `PlaywrightRecordingMetadata` since it's coming from a replay package or maybe `PlaywrightReplayMetadata` if we wanted to keep `Replay` in the name.